packages/cashier/src/components/error-dialog/error-dialog.tsx

```diff
@@ -13,9 +13,9 @@ type TErrorDialogProps = {
 
 type TSetDetails = {
     title: string;
-    cancel_button_text: undefined | string;
+    cancel_button_text?: undefined | string;
```

Comment:
since you've made it optional there should be no need for the undefined anymore.
```suggestion
    cancel_button_text?: string;
```